.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,15 @@
+name: lint_python
+on: [push, pull_request]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
```

Comment:
Could we make this debian?

Reply:
No.

Reply:
GitHub runners support Ubuntu, Windows, and macOS. https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners